server/heroes/heroCtrl.js

```diff
@@ -62,10 +62,10 @@ module.exports = {
 
       // use environment variable in heroku if deployed, api_keys.js if local
       var yelp = Yelp.createClient({
-        consumer_key: process.env.YELP_CONSUMER_KEY || Auth.yelp.consumer_key,
-        consumer_secret: process.env.YELP_CONSUMER_SECRET || Auth.yelp.consumer_secret,
-        token: process.env.YELP_TOKEN || Auth.yelp.token,
-        token_secret: process.env.YELP_TOKEN_SECRET || Auth.yelp.token_secret
+        consumer_key: process.env.YELP_CONSUMER_KEY || api_keys.consumer_key,
+        consumer_secret: process.env.YELP_CONSUMER_SECRET || api_keys.consumer_secret,
+        token: process.env.YELP_TOKEN || api_keys.token,
+        token_secret: process.env.YELP_TOKEN_SECRET || api_keys.token_secret
       });
```

Comment:
I can make a new file with all the code and then `require` that file at each controller. I agree it's better code structure. I can work on it later, right now the code is not allowing me to deploy.